packages/material-ui/src/MenuList/MenuList.test.js

```diff
@@ -45,4 +45,76 @@ describe('<MenuList />', () => {
       assert.strictEqual(wrapper.find('div').length, 2);
     });
   });
+
+  describe('actions: adjustStyleForScrollbar', () => {
+    it('should not adjust style when container element height is greater', () => {
+      const menuListActionsRef = React.createRef();
+      const wrapper = mount(<MenuList actions={menuListActionsRef} />);
+      const list = wrapper.getDOMNode();
+      assert.strictEqual(list.style.paddingRight, '');
+      assert.strictEqual(list.style.paddingLeft, '');
+      assert.strictEqual(list.style.width, '');
+      menuListActionsRef.current.adjustStyleForScrollbar(
+        { clientHeight: 10 },
+        { direction: 'ltr' },
+      );
+      assert.strictEqual(list.style.paddingRight, '');
+      assert.strictEqual(list.style.paddingLeft, '');
+      assert.strictEqual(list.style.width, '');
+    });
+
+    it('should adjust style when container element height is less', () => {
+      const menuListActionsRef = React.createRef();
+      const wrapper = mount(<MenuList actions={menuListActionsRef} />);
+      const list = wrapper.getDOMNode();
+      Object.defineProperty(list.style, 'width', { writable: true, value: '' });
```

Comment:
Would
```suggestion
      stub(list.style, 'width').value('');
```
this work as well? Might throw if the `width` doesn't exist yet. Otherwise I'm not sure `defineProperty` is very robust.

Reply:
@oliviertassinari or @eps1lon I'm having difficulty getting Chrome 41 Karma tests to succeed mainly because I can't reproduce the problem locally (when I run `yarn test:karma` all the tests succeed but with a different list of browsers than ci) so I'm just making guesses about fixes. The `defineProperty` approach works well for jsdom (was recommended in the jsdom documentation), but I've tried a few different approaches to get the browser tests working without success. I would appreciate it if someone else could tackle this or if you can direct me on how to reproduce this on my windows machine I'd be happy to troubleshoot it further.

Reply:
@oliviertassinari and @eps1lon Nevermind my previous comment. I did some reading into Karma and found the browserstack information. I have a browserstack account, so I've now managed to execute the Karma tests with the browserstack browsers reproducing the CI failure. I'll be able to troubleshoot it on my own now.